pegasus/sites.v3/code.org/views/lms/lms_partner_top_header.haml

```diff
@@ -0,0 +1,10 @@
+.wrapper.grid-container.justify-space-between.align-items-center.gap-4
+  .text-wrapper
+    %h1.white
+      =hoc_s("lms_partner_page.heading", locals: {website: provider})
+    %p.white
+      =hoc_s("lms_partner_page.desc", locals: {website: provider})
+    %a.link-button.has-external-link{href: '#integration'}
```

Comment:
Remove the external link icon and make this white:
```suggestion
    %a.link-button.white{href: '#integration'}
```

Reply:
resolved thanks!